src/product/security.md

```diff
@@ -86,22 +103,30 @@ It's recommended to run the FlowFuse application and the flow runtimes on
 different domains. This will create separation of concerns and enhances security
 for, among others, cookies.
 
-_FlowFuse Cloud_: Our SaaS offering has multiple service domains. The
-corporate website is hosted at [flowforge.com](https://flowforge.com). The
-application is served at [app.flowfuse.com]({{ site.appURL }}), while
-the Node-RED instances are served from \*.flowforge.cloud.
+_FlowFuse Cloud_: Our SaaS offering has multiple domains:
+
+| domain | purpose |
+|--------|---------|
+|`app.flowfuse.com` | The FlowFuse Cloud platform |
+|`mqtt.flowfuse.com`| The MQTT endpoint for the platform - via secure WebSocket connection |
+|`*.flowfuse.cloud` | The endpoints of individual Node-RED instances |
+|`*.flowforge.cloud` | Older Node-RED instances may be served under this domain as they were created prior to the migration to the FlowFuse name |
+
+When running the FlowFuse Device Agent on a user's own hardware and network, it will require access to the following domains to be fully operational:
 
-MQTT traffic is served from mqtt.flowforge.cloud as MQTT over WebSockets.
+ - `app.flowfuse.com`
+ - `mqtt.flowfuse.com`
+ - `registry.npmjs.org` - to enable installing of Node-RED modules
```

Comment:
Possibly add bit about `catalogue.nodered.org` (and our custom catalogue server)

Reply:
I left that off because the Device Agent doesn't need access to it - it is loaded by the browser when the user is accessing the editor.

Reply:
OK